var/spack/repos/builtin/packages/stat/package.py

```diff
@@ -59,20 +61,18 @@ class Stat(Package):
 
     patch('configure_mpicxx.patch', when='@2.1.0')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            "--enable-gui",
```

Comment:
The GUI is enabled by default and needs to be explicitly disabled if one doesn't want it. I may create a variant for that, but it would require a fix to STAT's build system.